libs/designsystem/src/lib/scss/print/components/_ion-tabbar.scss

```diff
@@ -0,0 +1,5 @@
+@media print {
+  ion-tab-bar {
```

Comment:
Is this used to target the [`kirby-tabs`](https://cookbook.kirby.design/home/showcase/tabs) component? If that is the case consider either adding a comment stating so or rename the file to reflect that it exists to target tabs component.